cluster/ci/config/prow/openshift/cherrypick.yaml

```diff
@@ -50,6 +50,15 @@ items:
             readOnly: true
           - name: tmp
             mountPath: /tmp
+          resources:
+            requests:
+              memory: "3Pi"
+              cpu: "250m"
+              ephemeral-storage: "2Mi"
+            limits:
```

Comment:
I don't think we want limits, just requests.

Reply:
ok let me remove them